zetaclient/ratelimiter/rate_limiter_test.go

```diff
@@ -115,13 +115,15 @@ func Test_ApplyRateLimiter(t *testing.T) {
 	// define test chain ids
 	ethChainID := chains.EthChain.ChainId
 	btcChainID := chains.BtcMainnetChain.ChainId
+	zetaChainID := chains.ZetaTestnetChain.ChainId
```

Comment:
We're using mainnet external chains but ZetaChain testnet, this seems inconsistent

Reply:
https://github.com/zeta-chain/node/pull/2256/commits/d6de55cb052850aea11bcb8e8a7ef0e8fc4398e9